<launch>
    <!-- ################################ ACTION SERVER ################################ -->
    <node pkg="graph_d_exploration" type="server_merge.py" name="merge_points" output="screen"/>

    <!-- ################################ ACTION SERVER ################################ -->
    <node pkg="graph_d_exploration" type="server_choose.py" name="choose_points" output="screen"/>

    <!-- ################################ MANAGER ################################ -->
    <node pkg="graph_d_exploration" type="manager.py" name="manager" output="screen">
        <param name="num_robots" value="2"/>
    </node>
</launch>